tap_snapchat_ads/__init__.py

```diff
@@ -32,9 +34,19 @@ def main():
 
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
 
+    request_timeout = parsed_args.config.get('request_timeout')
+
+    # if request_timeout is other than 0, "0" or "" then use request_timeout
+    if request_timeout and float(request_timeout):
+        request_timeout = float(request_timeout)
+    else: # If value is 0, "0" or "" then set the default which is 300 seconds.
+        request_timeout = REQUEST_TIMEOUT
```

Comment:
Move this calculation to the init function of client.py to keep consistency.

Reply:
All the calculations related to request_timeout moved to the client.py